stages/infra-setup/TCID-DIR-INSTALL-ON-LOCAL-HP

```diff
@@ -41,22 +43,51 @@ node() {
   # Create clusterrolebinding
   kubectl create clusterrolebinding kube-admin --clusterrole cluster-admin --serviceaccount=kube-system:default
 
+
+  # Deploy Kubera using either kubera-charts repo or Kubera official charts
+
+if [ "$USE_KUBERA_REPO" == "true" ]
+then
+
+  # Deploy Kubera from kubera-charts repo
+  echo -e "\nDeploying Kubera using kubera-charts repo\n"
+  echo -e "\nCloning $KUBERA_BRANCH branch of kubera-charts repo\n"
+  git clone -b $KUBERA_BRANCH https://$GITHUB_USERNAME:$GITHUB_PASSWORD@github.com/mayadata-io/kubera-charts.git
+  cd kubera-charts
+
+  echo -e "\nUpdating values in kubera-charts/values.yaml\n"
+  # Update release version
+  sed 's|release: .*|release: '$RELEASE'|' -i ./values.yaml
+  # Update URL for accessing Kubera
+  sed 's|url: .*|url: '$DOP_URL'|' -i ./values.yaml
+  # Update max member count in a project to 100
+  sed 's|maxMemberCountInOneProject: .*|maxMemberCountInOneProject: 100|' -i ./values.yaml
+  # Enable openebs RC builds deployment
+  sed 's|openebsRCEnable: .*|openebsRCEnable: true|' -i ./values.yaml
+  # Disable nginx-ingress deployment as it is predeployed in Rancher
+  ## Sed command to replace the third line next to "nginx-ingress:"
+  sed '/nginx-ingress:/!b;n;n;n;c\  enabled: false' -i ./values.yaml
+  echo -e "\nInstalling Kubera using kubera-charts/values.yaml"
+  helm install --name kubera .
+
+  cd ..
+
+else
+
+  # Deploy Kubera using official Kubera charts
+  echo -e "\nDeploying Kubera using official Kubera charts\n"
   helm repo update
-  
   # Add kubera in local repository
   helm repo add kubera https://charts.mayadata.io/
-
   # Apply helm chart
-  helm install --name kubera kubera/kubera-charts --set type.installOpenebs=false --set server.url=$DOP_URL --set server.release=$RELEASE --set server.maxMemberCountInOneProject=100
-
-  # Pass these values as well if need to run ingress as deployment:  --set nginx-ingress.controller.kind=Deployment --set nginx-ingress.controller.service.enabled=true
-
+  helm install --name kubera kubera/kubera-charts --set server.url=$DOP_URL --set server.release=$RELEASE --set server.maxMemberCountInOneProject=100 --set nginx-ingress.enabled=false --set server.openebsRCEnable=true
```

Comment:
Remove `--set nginx-ingress.enabled=false` and add `--set type.installOpenebs=false`